v03_pipeline/lib/tasks/exports/write_new_clinvar_variants_parquet.py

```diff
@@ -0,0 +1,58 @@
+import hail as hl
+import luigi
+import luigi.util
+
+from v03_pipeline.lib.paths import (
+    new_clinvar_variants_parquet_path,
+    new_variants_table_path,
+)
+from v03_pipeline.lib.tasks.base.base_loading_run_params import (
+    BaseLoadingRunParams,
+)
+from v03_pipeline.lib.tasks.base.base_write_parquet import BaseWriteParquetTask
+from v03_pipeline.lib.tasks.exports.misc import (
+    unmap_reference_dataset_annotation_enums,
+)
+from v03_pipeline.lib.tasks.files import GCSorLocalTarget
+from v03_pipeline.lib.tasks.update_new_variants_with_caids import (
+    UpdateNewVariantsWithCAIDsTask,
+)
+from v03_pipeline.lib.tasks.write_new_variants_table import WriteNewVariantsTableTask
+
+
+@luigi.util.inherits(BaseLoadingRunParams)
+class WriteNewClinvarVariantsParquetTask(BaseWriteParquetTask):
```

Comment:
just confirming that, per our discussion, this is not the approach we are going to be using for MVP. Okay to leave in for now and take out later when the real approach is implemented